clients/node/client-rds-data-node/protocol/AwsRestJson1_1.ts

```diff
@@ -83,49 +88,36 @@ export function executeStatementAwsRestJson1_1Deserialize(
 function executeStatementAwsRestJson1_1DeserializeError(
   output: HttpResponse
 ): Promise<ExecuteStatementResponse> {
-  let data = parseBody(output);
-  if (output.statusCode === 400 && data.dbConnectionId !== undefined) {
-    return Promise.reject({
-      __type: "com.amazon.rdsdataservice#StatementTimeoutException",
-      $name: "StatementTimeoutException",
-      $fault: "client",
-      message: data.message,
-      dbConnectionId: data.dbConnectionId
-    });
-  }
-
-  if (output.statusCode === 400) {
-    return Promise.reject({
-      __type: "com.amazon.rdsdataservice#BadRequestException",
-      $name: "BadRequestException",
-      $fault: "client",
-      message: data.message
-    });
-  }
-
-  if (output.statusCode === 403) {
-    return Promise.reject({
-      __type: "com.amazon.rdsdataservice#ForbiddenException",
-      $name: "ForbiddenException",
-      $fault: "client",
-      message: data.message
-    });
-  }
-
-  if (output.statusCode === 500) {
-    return Promise.reject({
-      __type: "com.amazon.rdsdataservice#InternalServerErrorException",
-      $name: "InternalServerErrorException",
-      $fault: "server"
-    });
-  }
+  const data = parseBody(output);
```

Comment:
parseBody return you a promise. Need to wait here

Reply:
updated in 02fb1f0be6e0789efab87708333f00e8422d32ce